docs-src/concepts/what-is-remote-data-encoding.md

```diff
@@ -0,0 +1,36 @@
+---
+id: what-is-remote-data-encoding
```

Comment:
I would rename it all to Remote Data Encoder. Not encoding. There is nothing special in the process here. There is something special in the infrastructure piece. So we should be talking about an Encoder, not Encoding.